include/umf/pools/pool_disjoint.h

```diff
@@ -16,68 +16,92 @@ extern "C" {
 ///        i.e. if multiple pools use the same shared limits, sum of those pools'
 ///        sizes cannot exceed MaxSize.
 typedef struct umf_disjoint_pool_shared_limits_t
-    umf_disjoint_pool_shared_limits_t;
+    *umf_disjoint_pool_shared_limits_handle_t;
 
-/// @brief Create a pool limits struct
-/// @param MaxSize specifies hard limit for memory allocated from a provider
-/// @return pointer to created pool limits struct
-umf_disjoint_pool_shared_limits_t *
+struct umf_disjoint_pool_params_t;
+/// @brief handle to the parameters of the disjoint pool.
+typedef struct umf_disjoint_pool_params_t *umf_disjoint_pool_params_handle_t;
+
+/// @brief Create a pool limits struct.
+/// @param MaxSize specifies hard limit for memory allocated from a provider.
+/// @return handle to the created shared limits struct.
```

Comment:
should we mention default values of all params?

Reply:
I think it is better to be done in a separate PR. We need to agree on default values. I have created the issue for that #915